ocaml/xapi-cli-server/dune

```diff
@@ -11,7 +11,7 @@
    result
    rresult
    sexplib
-   tar-unix
+   tar
```

Comment:
are we missing a dependency on `xapi-aux` here?

```
[ben@piglet:xapi-cli-server {stockholm-test} ((HEAD detached at psafont/tar))]$ rg Tar
<..>
cli_operations.ml:                    Tar_helpers.write_end sock;
```

Reply:
xapi_aux is on line 24 ;)